src/ZenifyCodingStandard/Sniffs/WhiteSpace/OperatorSpacingSniff.php

```diff
@@ -78,7 +78,7 @@ public function process(PHP_CodeSniffer_File $file, $position)
 		$isSpaceAfter = ($this->tokens[$position + 1]['code'] === T_WHITESPACE);
 		$isNewlineAfter = ($this->tokens[$position]['line'] !== $this->tokens[$position + 2]['line']);
 
-		if ( ! $isSpaceBefore || ! $isSpaceAfter || $isNewlineAfter) {
+		if ( ! $isSpaceBefore || ! ( $isSpaceAfter || $isNewlineAfter ) ) {
```

Comment:
There is no sniff for this yet, but there are only spaces around `!`, not inside `()`.

This is correct then:
```php
`if ( ! $isSpaceBefore || ! ($isSpaceAfter || $isNewlineAfter)) {
```